src/popup/router/routes.js

```diff
@@ -7,6 +7,7 @@ import SeedPhraseComponent from './pages/SeedPhrase';
 import TransactionsComponent from './pages/Transactions';
 import TransactionDetailsComponent from './pages/TransactionDetails';
 import SignTransactionComponent from './pages/SignTransaction';
+import ManageAccount from './pages/manageAccounts';
```

Comment:
`import ManageAccount from './pages/ManageAccounts';`

Please fix this. (capital letter M).

It is preventing the successful loading of the extension.

Reply:
Fixed